# Goal
uint64 count_until
---
# Result
string message
---
# Feedback
uint64 current_value
